install/kubernetes/helm/istio/charts/meshexpansion/templates/config-envoy.yaml

```diff
@@ -0,0 +1,181 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: meshexpansion-envoy-config
+  namespace: istio-system
+data:
+  envoy.yaml: |
+    stats_config:
+      use_all_default_tags: false
+
+    admin:
+      access_log_path: /dev/stdout
+      address:
+        # Only allow debugging from localhost
+        socket_address: { address: 127.0.0.1, port_value: 15000 }
+
+    dynamic_resources:
+      lds_config: {"ads": {}}
+      cds_config: {"ads": {}}
+      ads_config:
+        api_type: GRPC
+        refresh_delay: 10s
+        cluster_names: [xds-grpc]
+
+    static_resources:
+      listeners:
+
+      - name: tcp-zvpn-pilot
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 15010 }
+        filter_chains:
+        - filters:
+          - name: envoy.tcp_proxy
+            config:
+              deprecated_v1: true
+              value:
+                stat_prefix: tcp_pilot
+                route_config:
+                    routes:
+                    - cluster: zvpn.pilot
```

Comment:
You don’t need deprecated v1, if it’s just a single cluster with no selection. Use the cluster parameter in v2 tcp filter